tool/randomizer.lua

```diff
@@ -20,44 +20,81 @@ dependencies.beginner = {
     [22] = { 3, 11, 12, 20, 30 },
 }
 
+randomizer.SEEDTYPE = {
+    RANDOMSEED = "Random Seed",
+    SETSEED = "Set Seed",
+}
+
 local dependants = {}
-for proficiency, deps in pairs(dependencies) do
-    dependants[proficiency] = {}
+for difficulty, deps in pairs(dependencies) do
+    dependants[difficulty] = {}
     for level, requirements in pairs(deps) do
         for _, requirement in ipairs(requirements) do
-            local list = dependants[proficiency][requirement] or {}
+            local list = dependants[difficulty][requirement] or {}
             table.insert(list, level)
-            dependants[proficiency][requirement] = list
+            dependants[difficulty][requirement] = list
         end
     end
 end
 
-randomizer.seed = ""
-randomizer.proficiency = "beginner"
-randomizer.proficiencies = { "beginner", "intermediate", "advanced" }
+-- FALSE when the hud is displaying info about the current seed, TRUE when it's displaying info about the next game's seed
+randomizer.hudisfornextseed = true
```

Comment:
Isn't this variable redundant? If `seedqueue[1]` always indicates the current seed, and `seedqueue[2]` always indicates the next seed, shouldn't it be enough for `seedqueue[1]` to just be `nil`, which would be equivalent to this variable?